cltk/tests/test_phonology.py

```diff
@@ -486,6 +487,123 @@ def test_old_norse_transcriber(self):
                  "tvaː ɛr ɛːtːir ɛru fraː kɔmnar adam ɔk ɛvu ɔk fjœlɣaðist θɛira kynsloːð ɔk drɛivðist um hɛim alːan]"
         self.assertEqual(target, transcribed_sentence)
 
+    def test_utils(self):
```

Comment:
Break this up in more tests (e.g. break_utils_consonants, break_utils_vowels ...)